src/math.rs

```diff
@@ -364,44 +367,46 @@ where
     }
 
     fn contains(&self, p: &Point3<S>) -> bool {
-        let Point3 { x, y, z } =
-            self.isometry_inv.rotation.rotate_point(*p) + self.isometry_inv.translation;
+        let Point3 { x, y, z } = self.to_obb.rotation.rotate_point(*p) + self.to_obb.translation;
         x.abs() <= self.half_extent.x
             && y.abs() <= self.half_extent.y
             && z.abs() <= self.half_extent.z
     }
 
     fn transform(&self, isometry: &Isometry3<S>) -> Box<dyn PointCulling<S>> {
-        Box::new(Self::new(isometry * &self.isometry, self.half_extent))
+        Box::new(Self::new(isometry * &self.from_obb, self.half_extent))
     }
 }
 
 #[derive(Debug, Clone)]
 pub struct OrientedBeam<S> {
     // The members here are an implementation detail and differ from the
     // minimal representation in the gRPC message to speed up operations.
-    // Isometry_inv is the transform from world coordinates into "beam coordinates".
-    isometry: Isometry3<S>,
-    isometry_inv: Isometry3<S>,
+    // to_beam is the transform from world coordinates into "beam coordinates".
+    from_beam: Isometry3<S>,
+    to_beam: Isometry3<S>,
     half_extent: Vector2<S>,
     corners: [Point3<S>; 4],
     separating_axes: Vec<Vector3<S>>,
 }
 
 impl<S: BaseFloat> OrientedBeam<S> {
-    pub fn new(isometry: Isometry3<S>, half_extent: Vector2<S>) -> Self {
+    pub fn new(from_beam: Isometry3<S>, half_extent: Vector2<S>) -> Self {
         OrientedBeam {
-            isometry_inv: isometry.inverse(),
+            to_beam: from_beam.inverse(),
             half_extent,
-            corners: OrientedBeam::precompute_corners(&isometry, &half_extent),
-            separating_axes: OrientedBeam::precompute_separating_axes(&isometry.rotation),
-            isometry,
+            corners: OrientedBeam::precompute_corners(&from_beam, &half_extent),
+            separating_axes: OrientedBeam::precompute_separating_axes(&from_beam.rotation),
+            from_beam,
         }
     }
 
-    fn precompute_corners(isometry: &Isometry3<S>, half_extent: &Vector2<S>) -> [Point3<S>; 4] {
+    fn precompute_corners(from_beam: &Isometry3<S>, half_extent: &Vector2<S>) -> [Point3<S>; 4] {
```

Comment:
maybe here it is worth mentioning that the "corners"  are defined as the intersection of the beam with z = 0

Reply:
Done.